freud/plot.py

```diff
@@ -30,10 +42,12 @@ def bar_plot(x, height, title=None, xlabel=None, ylabel=None, ax=None):
         title (str): Title of the graph. (Default value = :code:`None`).
         xlabel (str): Label of x axis. (Default value = :code:`None`).
         ylabel (str): Label of y axis. (Default value = :code:`None`).
+        ax (:class: matplotlib.Axes.axes): axes object to plot on.
+            If :code:`None`, make a new axes and figure object.
+            (Default value = :code:`None`).
 
     Returns:
-        bytes: Byte representation of the graph in png format if matplotlib is
-        available. Otherwise :code:`None`.
+        :class: matplotlib.Axes.axes: axes object with the diagram.
```

Comment:
I think the class is `matplotlib.axes.Axes` ([docs](https://matplotlib.org/3.1.0/api/axes_api.html#matplotlib.axes.Axes)). To render this correctly, we need to do two things:
1. Fix the syntax everywhere to
```python
:class:`matplotlib.axes.Axes`
```
2. Add `matplotlib` to our documentation via [intersphinx](http://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html). See the bottom of `doc/source/conf.py`.